### PR TITLE
chore: polish voting

### LINKED ISSUE #2299: [Task]: Polish voting UI
### Task description

Polish voting UI/UX to be production ready

### Requirements

- Lightmode/Darkmode
- Responsivity with different screen resolutions, languages and sizes
- Looking like figma layouts
- Defensive coding/layout (don't break the app or UX for the user)
- Test edge cases

### Acceptance criteria

Production ready

### Creation checklist

- [ ] I have assigned this task to the correct people
- [ ] I have added the most appropriate labels
- [ ] I have linked the correct milestone and/or project

### PR DESCRIPTION
**Pull request by jeeanribeiro**
## Summary
- Dark/light theme fixes
- Responsive layout fixes
- Not found treasury event state layout and working buttons
- Voting button's layout fixes
- i18n to strings in governance event details
- Removes wallet pill
- Makes "Learn about voting with Firefly" link to work (added on Electron URL whitelist)

## Relevant Issues
closes #2299

## Type of Change
Please select any type below that applies to your changes, and delete those that are not.
- [ ] __Breaking__ - any change that would cause existing functionality to not work as expected
- [x] __Chore__ - refactoring, build scripts or anything else that isn't user-facing
- [ ] __Docs__ - changes to the documentation
- [ ] __Fix__ - a change which fixes an issue
- [ ] __New__ - a change which implements a new feature
- [ ] __Update__ - a change which updates existing functionality

### Platforms
Please select any platforms where your changes have been tested.
- __Desktop__
	- [ ] MacOS
	- [x] Linux
	- [ ] Windows
- __Mobile__
	- [ ] iOS
	- [ ] Android

### Instructions
Test everything in governance flow with attention to details

## Checklist
Please tick all of the following boxes that are relevant to your changes, and delete those that are not.
- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have added or modified tests that prove my changes work as intended
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have verified that my latest changes pass CI workflows for testing and linting
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/shared/routes/dashboard/governance/views/GovernanceInfo.svelte [hunk 1]
```diff
@@ -5,18 +5,21 @@
...
         >
     </div>
     <Link
-        classes="mr-5 text-13 text-gray-800 border border-solid border-gray-300 rounded-xl py-4 pl-6 flex justify-between bg-gray-50 hover:bg-blue-100"
+        classes="mr-5 text-13 text-gray-800 dark:text-white border border-solid border-gray-300 dark:border-transparent
+        py-4 pl-6 flex justify-between bg-gray-50 hover:bg-blue-100 dark:hover:bg-gray-700 dark:bg-gray-900 rounded-xl w-full h-fit"
         onClick={() => Electron.openUrl('https://iotatreasury.org/how-voting-works.html')}
     >
-        {localize('views.governance.info.link')}
+        <Text type="p" classes="truncate">{localize('views.governance.info.link')}</Text>
```
Comment (Tuditi) [inline packages/shared/routes/dashboard/governance/views/GovernanceInfo.svelte:22]: Same problem as with other button. Text should be centered.

**Review by Tuditi — CHANGES_REQUESTED** [review]
I left some comments, but great work in general. UI is starting to look really good!

File: packages/shared/routes/dashboard/governance/views/GovernanceEvents.svelte [hunk 2]
```diff
@@ -1,29 +1,80 @@
...
+            >
+        </div>
+        {#if !networkConfig.automaticNodeSelection}
+            <Button onClick={handleConnectDefaultNodeClick} classes="mb-2">
+                <Text type="h5" bold classes="text-white px-4" overrideColor
+                    >{localize('views.governance.events.treasury.notFound.buttonConnect')}
+                </Text>
+            </Button>
+        {/if}
+        <Button secondary onClick={handleViewNodeSettingsClick} classes="border-transparent">
```
[the remark this answers is not in the document]
Comment (jeeanribeiro) [reply packages/shared/routes/dashboard/governance/views/GovernanceEvents.svelte:74]: I agree, I've pinged them in Discord, would be better if we change the text to be in the center

[the remark this answers is not in the document]
Comment (jeeanribeiro) [reply packages/shared/routes/dashboard/governance/views/GovernanceEvents.svelte:74]: But we could do it as enhancement and not block this task

[the remark this answers is not in the document]
Comment (Tuditi) [reply packages/shared/routes/dashboard/governance/views/GovernanceEvents.svelte:74]: Very true, I created an issue for it.
